emcc.py

```diff
@@ -1431,8 +1431,14 @@ def phase_setup(options, state, newargs, user_settings):
   if settings.USE_PTHREADS or settings.WASM_WORKERS:
     settings.SHARED_MEMORY = 1
 
-  if settings.SHARED_MEMORY and '-pthread' not in newargs:
-    newargs += ['-matomics', '-mbulk-memory']
+  if settings.SHARED_MEMORY:
```

Comment:
I think this should probably in the `#elif` bock after `settings.USE_PTHREADS` below?